mainapp/views.py

```diff
@@ -6,49 +8,41 @@
 # Create your views here.
 
 def home(request):
-    count = User.objects.count()
-    return render(request, 'mainapp/home.html', {
-        'count' : count
-        })
+    count: Any = User.objects.count()
```

Comment:
Don't add `Any` typing to un-typed functions. Make all object (including attributes) are typed or add docstring.

Reply:
See Example:

```python3
from django.http import HttpRequest, Responce 

def home(request: HttpRequest) -> Responce:
    count: int = User.objects.count()  # That's count and it always int
    return render(request, 'mainapp/home.html', {'count': count})

```

Reply:
That's hasn't no sense because Django hasn't build-ins typing.